### PR TITLE
Pract.03: Add Task01 | Manoela Barakova

### PR DESCRIPTION
**Pull request by Manoela2003**
Manoela Barakova
SI 5 5MI0600271

### CODE REVIEW

**Comment by Justsvetoslavov** [comment]
Прочети тук: https://github.com/Justsvetoslavov/Object-oriented_programming_FMI/tree/master/Github-Workflow и редактирай PR-a.

File: Seminars/Sem.03/Pract.03/Task01.cpp [hunk 1]
```diff
@@ -1,102 +1,241 @@
...
+		std::cout << "The file already has 1024 students!";
+		return;
+	}
+	else if (countOfSavedStudents + countOfUnsavedStudents > max_Count_Of_Students) {
+		std::cout << "The maximum count of students you can enter is " << max_Count_Of_Students - countOfSavedStudents << '\n';
+		countOfUnsavedStudents = max_Count_Of_Students - countOfSavedStudents;
+	}
+}
+
+void PrintHairColor(int hairIndex) {
```
Comment (dejansvetoslavov) [inline Seminars/Sem.03/Pract.03/Task01.cpp]: Why don't you use the enum?

File: Seminars/Sem.03/Pract.03/Task01.cpp [hunk 2]
```diff
@@ -1,102 +1,241 @@
...
+	UnsetCharSymbols(buffer);
+	file.getline(buffer, 5, ',');
+	st.averageScore = strtod(buffer, nullptr);
+	UnsetCharSymbols(buffer);
+	file.getline(buffer, 2, '\n');
+	CharToHairColor(st, buffer);
+	return st;
+}
+
+void CheckMaxSizeOfStudents(std::ifstream& file, unsigned& countOfUnsavedStudents, const int countOfSavedStudents) {
```
Comment (dejansvetoslavov) [inline Seminars/Sem.03/Pract.03/Task01.cpp]: unused

File: Seminars/Sem.03/Pract.03/Task01.cpp [hunk 3]
```diff
@@ -1,102 +1,241 @@
...
+		student.color = hairColor::brown;
+	else if (buffer[0] == '1')
+		student.color = hairColor::black;
+	else if (buffer[0] == '2')
+		student.color = hairColor::blonde;
+	else if (buffer[0] == '3')
+		student.color = hairColor::red;
+	else
+		student.color = hairColor::white;
+}
```
Comment (dejansvetoslavov) [inline Seminars/Sem.03/Pract.03/Task01.cpp]: How can you use the enum here?

File: Seminars/Sem.03/Pract.03/Task01.cpp [hunk 4]
```diff
@@ -1,102 +1,241 @@
...
+	Student st;
+	char buffer[10];
+	file.getline(st.firstName, max_Name_Size + 1, ',');
+	file.getline(st.familyName, max_Name_Size + 1, ',');
+	file.getline(buffer, 6, ',');
+	st.facultyNumber = CharToInt(buffer);
+	UnsetCharSymbols(buffer);
+	file.getline(buffer, 5, ',');
+	st.averageScore = strtod(buffer, nullptr);
+	UnsetCharSymbols(buffer);
```
Comment (dejansvetoslavov) [inline Seminars/Sem.03/Pract.03/Task01.cpp]: Why is this needed?
